### DISCUSSION TITLE
FDS source code of FDS6.1.0

### DISCUSSION THREAD

**Discussion by lihy213** [firemodels/fds#13486] [Q&A]
Dear developers,

Could you please provide the source code of FDS 6.1.0, I can't find it and its hash code in the release version.

Kindly regards,

LI

**Comment by drjfloyd** [comment]
Prior to 6.3 FDS was hosted on GoogleCode which now no longer exists.  The executable for 6.1 can be found on the releases page.  GoogleCode used Subversion instead of Git so there isn't a git tag for versions from GoogleCode. If you clone the fds repo, you can search the log file for 6.1.0. There should be a commit where we change the version number to 6.1.0
